torch/onnx/_internal/fx/export.py

```diff
@@ -21,9 +22,31 @@
 from torch.fx.passes import fake_tensor_prop
 from torch.nn.utils import stateless
 from torch.onnx import _constants, _type_utils
-from torch.onnx._internal import _beartype
+
+from torch.onnx._internal import _beartype, diagnostics
 from torch.utils import _pytree
 
+
+class _GetAttrCallable(Protocol):
+    def __call__(self, name: str) -> Any:
+        ...
+
+
+def _module_not_available(module_name: str, package_name: str) -> _GetAttrCallable:
+    def _inner(name: str):
+        raise RuntimeError(
+            f"{module_name} is not available. Please install {package_name} to use this feature."
+        )
+
+    return _inner
+
+
+def _fake_module_import(module_name: str, package_name: str) -> ModuleType:
+    module = ModuleType(module_name)
+    module.__getattr__ = _module_not_available(module_name, package_name)
```

Comment:
is `_module_not_available` consumed somewhere else or have future usage. It seems like it's only a one line function. Consider not use the function?